snap/README.md

```diff
@@ -1,73 +1,179 @@
+The snap can be installed on any system that supports snaps. You can see how to install 
+snaps on your system [here](https://docs.snapcraft.io/t/installing-snapd/6735).
+
+However for full security confinement, the snap should be installed on an 
+Ubuntu 16.04 LTS or later Desktop or Server, or a system running Ubuntu Core 16 or later.
+
+### Installing EdgeX Foundry as a snap
+The snap is published in the snap store at https://snapcraft.io/edgexfoundry.
+You can see the current revisions available for your machine's architecture by running the command:
+
+```bash
+$ snap info edgexfoundry
+```
+
+The snap can be installed using `snap install`. To install the snap from the edge channel:
 
-The snap can be installed using this command:
+```bash
+$ sudo snap install edgexfoundry --edge
+```
 
-`$ sudo snap install edgexfoundry --channel=california/edge`
+You can specify install specific releases using the `--channel` option. For example to install the california release of the snap:
 
-**Note** - this snap has only been tested on Ubuntu 16.04 LTS Desktop/Server and Ubuntu Core 16.
+```bash
+$ sudo snap install edgexfoundry --channel=california
+```
 
-## Configuration
-The hardware-observe, process-control, mount-observe, and system-observe snap interfaces needs to be
+Lastly, on a system supporting it, the snap may be installed using GNOME (or Ubuntu) Software Center by searching for `edgexfoundry`.
+
+**Note** - the snap has only been tested on Ubuntu 16.04 LTS Desktop/Server and Ubuntu Core 16.
+
+### Configuration
+The `hardware-observe`, `process-control`, `mount-observe`, and `system-observe` snap interfaces needs to be
 connected after installation using the following commands:
 
-`$ snap connect edgexfoundry-core:hardware-observe core:hardware-observe`
+```bash
+$ sudo snap connect edgexfoundry:hardware-observe
+$ sudo snap connect edgexfoundry:process-control
+$ sudo snap connect edgexfoundry:system-observe
+$ sudo snap connect edgexfoundry:mount-observe
+```
 
-`$ snap connect edgexfoundry-core:process-control core:process-control`
+After connecting these restart the services in the snap with:
 
-`$ snap connect edgexfoundry-core:system-observe core:system-observe`
+```bash
+$ sudo snap restart edgexfoundry
+```
 
-`$ snap connect edgexfoundry-core:system-observe core:mount-observe`
+**Note** - these interface will be connected automatically after https://forum.snapcraft.io/t/edgexfoundry-auto-connections-assertions-request/7920/1 has been processed. After this is done, then connecting these interfaces will only be necessary during development. See the section below on building the snap for more details.
 
+**Note** - the process-control interface will be dropped after all of the services are supported as proper daemons in the snap.
 
-## Starting/Stopping EdgeX
-To start all the EdgeX microservices, use the following command:
+## Using the EdgeX snap
+To start all the EdgeX microservices, run the following command as root:
```

Comment:
I'd change to "as root" to "using sudo".

Reply:
Done